### ISSUE TITLE
Scaling should be transferred from training to test

### ISSUE THREAD

**Issue by Jakob-Bach** [CemOezcan/metalfi#36]
In `MetaModel`, training and test set are currently scaled independently, i.e., one scaler is trained for each data set. However, you should avoid operations which make use of several test objects at the same time -- once the (meta-)model/whole prediction pipeline is trained, it should be able to make predictions for each test object independently, which is not possible if a whole set of test objects is used for scaling. The proper way to do this is training the scaler on the training set and only applying it to the test set (`transform()` instead of `fit_transform`()).

**Comment by CemOezcan** [comment]
Fixed by #37

**Comment by Jakob-Bach** [comment]
The current fix introduced in 8bfd454 does not look correct to me, as it concatenates the test set to the training set before scaling, which results in using test information in the pipeline, which shouldn't be done.

**Comment by Jakob-Bach** [comment]
Fixed by 48b86eaf93
